openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -62,30 +62,41 @@ def get_settings_from_context(self, context):
             .get("publish", {})
             .get(self.__class__.__name__, {})
         )
-        get_from_context = False
+
+        settings_from_context = []
```

Comment:
sorry for having led you to this wrong data structure, but I think a `dict` would be more appropriate.